hummingbird/ml/_topology.py

```diff
@@ -39,11 +39,6 @@
 else:
     DataFrame = None
 
-if pandas_installed():
```

Comment:
This looks suspicious to me

Reply:
For some reason it github continue to present me this.

Reply:
Hahah, something very strange happened with this one function!
you need to add it back here, right? (since this is the only copy now since we deleted the duplicate in the prior PR?)